scriptExamples/extendedMode/python-logging/script-connection-based.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+"""
+Script to debug the "Scriptable Notifications" add-on for Thunderbird.
```

Comment:
Please add a comment saying that this script handles the "connection based" mode.

Reply:
You could also rename the other script "script.py" to "script-connectionless.py"?

Reply:
Done in last commit.